taxonomies/additives.txt

```diff
@@ -10323,29 +10741,48 @@ vegan:en: yes
 vegetarian:en: yes
 wikidata:en: Q18967194
 
-en: E386, E386 food additive
 xx: E386
-bg: E386, E386 food additive
-cs: E386, E386 food additive
-da: E386, E386 food additive
-de: E386, E386 food additive
-el: E386, E386 food additive
-es: E386, E386 food additive
-et: E386, E386 food additive
+ar: E386, إي دي تي إيه ثنائي الصوديوم
+az: E386, İki natrium EDTA
+bg: E386, Динатриев ЕДТА
+ca: E386, EDTA disòdic
+cs: E386, Disodná sůl EDTA
+da: E386, Dinatrium-EDTA
+de: E386, Dinatrium-EDTA
+el: E386, Δινάτριο EDTA
+eo: E386, Dinatria EDTA
+es: E386, EDTA disódico
```

Comment:
2025-03-07T21:50:07.0347344Z ERROR - duplicate_synonym - 12504 - es:edta-disodico already is a synonym of es:e385 for entry en:e385 (ingredients) - es:edta-disodico cannot be mapped to entry xx:e386 / es:e386

Reply:
fixed